### PR TITLE
fix: Responsiveness for Graduate Attributes in Contributor Page

### LINKED ISSUE #523: Responsiveness for Graduate Attributes in Contributor Page
### URL:

https://contributors.ohc.network/contributors/gigincg

### Browser Name:

Arc

### Browser Version:

Version 1.63.1 (54714)

### Operating System:

Mac

### How to reproduce the issue:

Graduate Attributes are not responsive on Mobile Screens; Remove the Horizontal Scroll and wrap the Sections as different rows

Image

### PR DESCRIPTION
**Pull request by VAIBHAVSING**
Fixes #523
fixed Responsiveness for Graduate Attributes in Contributor Page…  
not having any responsive issue with desktop or mobile
image
image

### CODE REVIEW

**Comment by gigincg** [comment]
@rithviknishad Can you get this reviewed

File: app/contributors/VAIBHAVSING.md
Comment (rithviknishad) [inline app/contributors/VAIBHAVSING.md:1]: The scraper will automatically add your profile upon first contribution to https://github.com/ohcnetwork/leaderboard-data

Could you discard this file?

**Review by rithviknishad — CHANGES_REQUESTED** [review]
Code LGTM; just remove the contributor profile file added

File: app/contributors/VAIBHAVSING.md
Comment (VAIBHAVSING) [reply app/contributors/VAIBHAVSING.md:1]: Can I remove and raise pr again

Comment (rithviknishad) [reply app/contributors/VAIBHAVSING.md:1]: you can just push in the same existing PR branch

**Review by rithviknishad — CHANGES_REQUESTED** [review]
Can we improve this a bit?

Just some ideas:

1. We could center align the group cards or make it `w-full` in mobile screens.
2. Align "Graduate Attributes" and "Learn more" with the card

Feel free to come up with more ideas if you have.

image

**Pull request merged**